core/src/main/scala/com/raphtory/spouts/WebSocketSpout.scala

```diff
@@ -20,20 +20,7 @@ import java.io.InputStreamReader
   * @param auth The authorization code in Base64 format to access the url e.g. "Basic XXXXXXXXX"
   * @param contentType The content type of the data e.g. application/json
   *
-  * @example
```

Comment:
why did u remove the example, its used in the docs to show how it works

Reply:
The code was old